src/aiperf/dataset/loader/hf_asr.py

```
@@ -0,0 +1,145 @@
+        and re-encode as WAV base64 in the format expected by the chat endpoint.
+        """
+        raw_bytes = audio_value.get("bytes")
+        if not raw_bytes:
+            return []
+        try:
+            array, sr = sf.read(io.BytesIO(raw_bytes))
+            buf = io.BytesIO()
+            sf.write(buf, array, sr, format="WAV")
+            b64 = base64.b64encode(buf.getvalue()).decode("utf-8")
+            return [Audio(name="", contents=[f"wav,{b64}"])]
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)